[Bug] [MASTER]  DependentExecute.java:398 NPE

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

3.2.0 upgrade to 3.2.1
```
[WI-0][TI-21145] - [ERROR] 2024-02-27 02:23:27.283 +0800 o.a.d.s.m.r.e.MasterTaskExecutor:[60] - Get a exception when execute the task, will try to cancel the task
java.lang.NullPointerException: null
        at org.apache.dolphinscheduler.server.master.utils.DependentExecute.getModelDependResult(DependentExecute.java:398)
        at org.apache.dolphinscheduler.server.master.utils.DependentExecute.finish(DependentExecute.java:364)
        at org.apache.dolphinscheduler.server.master.runner.task.dependent.DependentAsyncTaskExecuteFunction.isAllDependentTaskFinished(DependentAsyncTaskExecuteFunction.java:220)
        at org.apache.dolphinscheduler.server.master.runner.task.dependent.DependentAsyncTaskExecuteFunction.getAsyncTaskExecutionStatus(DependentAsyncTaskExecuteFunction.java:100)
        at org.apache.dolphinscheduler.server.master.runner.execute.AsyncMasterTaskDelayQueueLooper.lambda$run$0(AsyncMasterTaskDelayQueueLooper.java:90)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

初步判断是#14702 新增parameterPassing字段， 3.2.0版本的节点没有这个字段，升级到3.2.1版本DepenentItem.getParameterPassing()为null


### What you expected to happen

增加空值判断

### How to reproduce

3.2.0 upgrade to 3.2.1

### Anything else

_No response_

### Version

3.2.x

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/model/DependentItem.java
+++ b/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/model/DependentItem.java
@@ -35,7 +35,7 @@ public class DependentItem {
     private String dateValue;
     private DependResult dependResult;
     private TaskExecutionStatus status;
-    private Boolean parameterPassing;
+    private Boolean parameterPassing = false;
 
     public String getKey() {
         return String.format("%d-%d-%s-%s",

